nnef/src/lib.rs

```diff
@@ -33,5 +33,5 @@ pub mod internal {
 }
 
 pub fn nnef() -> framework::Nnef {
-    framework::Nnef::new()
+    framework::Nnef::new().with_tract_core()
```

Comment:
This is not desirable, we want to be able to process nnef "strictly", without allowing tract-core extensions. If you are dealing with loading a model from the command line, you can use `--nnef-tract-core` to enable the extensions.

Reply:
Thanks for clarification.